pkg/util/util.go

```diff
@@ -171,3 +172,37 @@ func GetNamespaceOrDefault(namespace string) string {
 	}
 	return namespace
 }
+
+func ParseClusterYaml(file string) (*clusterv1.Cluster, error) {
+	bytes, err := ioutil.ReadFile(file)
+	if err != nil {
+		return nil, err
+	}
+
+	cluster := &clusterv1.Cluster{}
+	err = yaml.Unmarshal(bytes, cluster)
```

Comment:
nit: this should be cuddled: `if err := yaml.Unmarshal(...); err != nil { ... }`